ruby/.gitignore

```diff
@@ -1,8 +1,21 @@
 *.bundle
 tags
 .idea/
+lib/google/protobuf/any_pb.rb
```

Comment:
Are these well known types added by intention?

Reply:
These were from my original PR if I recall. Those were auto generated files when you ran some subset of the tests locally.

Reply:
I think these should be added to .gitignore if you don't intentionally add them.

Reply:
Done.